doc/api/process.md

```diff
@@ -637,6 +637,28 @@ process's [`ChildProcess.disconnect()`][].
 If the Node.js process was not spawned with an IPC channel,
 `process.disconnect()` will be `undefined`.
 
+## process.dlopen(module, filename[, flags])
+<!-- YAML
+added: REPLACEME
```

Comment:
Gah, bad git-magic. I believe it was moved from `node.dlopen` to `process.dlopen` in ad0a4cefb8b4bf469bb1caa6e671a1009e40a81f, which made it to v0.1.16. Also v0.1.16 commit 726865af7bbafe58435986f4a193ff11c84e4bfe confirms it.